index.html

```diff
@@ -372,35 +293,24 @@ <h1>Contact</h1>
       </div>
     </div>
 
+    <!-- Insert single project modal -->
     <div id="single-project-modal" class="modal">
-      <span class="close">&times;</span>
+      <span class="close-popup">&times;</span>
```

Comment:
- Optional
I recommend changing the color of a button because most of the time it is disappearing on the white background. Like in this case. ⬇️. Also, when I click on the close popup button, I am taken back to the top of the app. Consider fixing it.
![image](https://user-images.githubusercontent.com/33268927/174312509-43764661-0d0b-47e1-aa6d-1bc5c1d442c2.png)